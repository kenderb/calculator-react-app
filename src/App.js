import React from 'react';

const App = () => <div>Hello from App component</div>;

export default App;
