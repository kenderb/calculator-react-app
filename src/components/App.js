import React, { Component } from 'react';

class App extends Component {
  state = {  }
  render() { 
    return ( 
      <div>Hello from App component</div>
    );
  }
}

export default App;