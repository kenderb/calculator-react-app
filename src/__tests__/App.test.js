import { create } from 'react-test-renderer';
import { render, screen, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom';
import App from '../components/App';

test('renders correctly', () => {
  const tree = create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
