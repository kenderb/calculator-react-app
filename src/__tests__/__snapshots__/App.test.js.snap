// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="sc-hKgILt iXAhNN"
  >
    <div
      className="sc-eCssSg eesNyB"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Math Magicians
      </a>
    </div>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  <main
    className="sc-pFZIQ lgBvhd"
  >
    <section
      className="sc-jSgupP jddVsL"
    >
      <h1>
        Welcome to our page!
      </h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitationullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitationullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
      </p>
    </section>
  </main>,
]
`;
